ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml

```diff
@@ -120,7 +101,7 @@ tests:
         requests:
           cpu: 100m
     workflow: optional-operators-ci-azure
-- as: e2e-infoblox-operator
+- as: e2e-azure-infoblox-operator
```

Comment:
This is currently failing in CI.

Reply:
Yes, all the tests were run but failed due to the expired temporary Infoblox licences. Let me recreate the Infoblox instance.

Reply:
[Done](https://github.com/openshift/release/pull/35668#issuecomment-1542932895).